### PR TITLE
Add topic design page

### LINKED ISSUE #10: Translate topic's Design page
Create page conteudo/topicos/design.md from https://github.com/brigadecore/brigade/blob/main/docs/content/topics/design.md

### PR DESCRIPTION
**Pull request by jorgearteiro**
Fixes #10 
Translation of a single page called design.md

### CODE REVIEW

File: conteudo/topicos/design.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+aconteceram após isso.
+
+No começo de 2020, Brigade se provou razoavelmenre popular entre entrevistados da
+[pesquisa da CNCF](https://www.cncf.io/wp-content/uploads/2020/08/CNCF_Survey_Report.pdf).
+Tendo sido popular, mantenedores tabém acumularam suficiente feedback da comunidade para
+reconhecer que algumas correcões seriam necessárias para que o projeto tivesse ainda mais
+sucesso. A maioria dos insights coletados mostraram a necessidade de encontrar uma melhor
+forma de abstrair os usuários do Brigade da complexidade do seu ambiente de execução, no
+caso o Kubernetes. Uma proposta formal para o Brigade V2 foi elaborada pelos mantenedores
+e ratificada pela comunidade. Nos últimos dois anos, Brigade foi re-escrito do zero. Brigade
```
Comment (lucasjesuss) [inline conteudo/topicos/design.md:199]:
```suggestion
e ratificada pela comunidade. Nos últimos dois anos, o Brigade foi re-escrito do zero. O Brigade
```

File: conteudo/topicos/design.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+v2 foi lançado em 1 de December de 2021.
```
Comment (lucasjesuss) [inline conteudo/topicos/design.md:200]:
```suggestion
v2 foi lançado em 1 de Dezembro de 2021.
```

**Review by lucasjesuss — COMMENTED** [review]
I left some adjustments
Any question, please, let me know

File: conteudo/topicos/design.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+Da mesma forma que o seu projeto irmão [Helm](https://helm.sh/), Brigade nasceu na Deis
+antes de sua eventual aquisição pela Microsoft. Ambos projetos foram resultado
+do mesmo processo e visualizam Kubernetes como um tipo de "sistema operacional
+do cluster". O time da Deis procurou identificar funcionalidades dos sistemas
+operacionais tradicionais que são muitas vezes consideradas padrão ou básicas,
+mas visivelmente não fazem parte do Kubernetes. Kubernetes não possui
+um gerenciador de pacotes e isso foi o que nos levou a criar o Helm, para cobrir
+este vazio no mercado. Da mesma forma, Kubernetes não possui um ambiente de
+scripting -- e este é o vazio que o Brigade foi desenhado para cobrir no mercado.
```
[the remark this answers is not in the document]
Comment (jorgearteiro) [reply conteudo/topicos/design.md:185]: This is ok, format will change that later.

File: conteudo/topicos/design.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+A imagem docker padrão do worker pode executar scripts escritos em
+[JavaScript](https://en.wikipedia.org/wiki/JavaScript) ou
+[TypeScript](https://www.typescriptlang.org/) usando
+[Node.js](https://nodejs.org/). Podendo também resolver qualquer dependência dos
+scripts usando
+[npm](https://nodejs.org/en/knowledge/getting-started/npm/what-is-npm/) ou
+[yarn](https://yarnpkg.com/).
+
+Através da configuração do projeto, é possível usar workers baseados numa imagem docker
+alternativa. Estas images poderiam prover suporte para manipuladores que são definidos
```
Comment (lucasjesuss) [inline conteudo/topicos/design.md:165]:
```suggestion
alternativa. Estas imagens poderiam prover suporte para manipuladores que são definidos
```

**Review by lucasjesuss — APPROVED** [review]

**Pull request merged**